yamale/command_line.py

```diff
@@ -20,7 +20,12 @@
 
 
 def _validate(schema_path, data_path, parser, strict):
-    schema = schemas.get(schema_path)
+    try:
+        if not os.path.getsize(schema_path):
+            raise ValueError('schema.yml is an empty file!')
+        schema = schemas.get(schema_path)
+    except Exception as e:
+        print(e)
```

Comment:
People can use Yamale as an API by calling `yamale.make_schema()` directly. Performing this check there may catch more uses than putting the check here.